docker/docker-compose.regtest.yml

```diff
@@ -0,0 +1,66 @@
+version: "2.2"
+
+services:
+    mongo-db:
+      image: "mongo"
+      restart: "no"
+      networks: [ "local" ]
+      ports:
+        - "27017"
+        - "27018"
+        - "27019"
+    mysql-db:
+      image: "mysql"
+      command: --default-authentication-plugin=mysql_native_password
+      restart:  "no"
+      environment:
+        MYSQL_ROOT_PASSWORD: infuy
+      networks: [ "local" ]
+      ports:
+      - "3306"
+      - "33060"
+    rsk-node:
+        image: "rsk-node-regtest-image:latest"
+        restart: "no"
+        networks: ["local"]
+        volumes:
+          - "./rsk-node/database:/var/lib/rsk/database/regtest"
+        ports:
+            - "4444"
+            - "30305"
+
+    lumino-explorer:
+# Not used because startExplorer has all the parameters already set
+#      environment:
+#        EXPLORER_PORT: 8080
+#        RSK_ENDPOINT: "http://rsk-node:4444"
+#        MONGO_DB_HOST: "mongo-db"
+#        TOKEN_NETWORK_REGISTRY: 0xed8c9163F888Bc2f9C4F299325003DA5fC8676DD
```

Comment:
since mainnet has this flag set, what about regtest? it uses a default one? or you must specify it here?

Reply:
Regtest is the default network so basically the scripts have default values using that.